dokka-runners/dokka-gradle-plugin/src/main/kotlin/engine/plugins/DokkaVersioningPluginParameters.kt

```diff
@@ -65,7 +68,7 @@ constructor(
      * For more information, see
      * [Directory structure](https://github.com/Kotlin/dokka/blob/master/plugins/versioning/README.md#directory-structure).
      */
-    @get:InputDirectory
+    @get:InputFiles
```

Comment:
Shall we add an explicit check that is is a directory?
From the mentioned Gradle issue: 
> What Gradle doesn't have is then some additional validation that if the file exists that it is actual a file/directory.